web/themes/custom/usagov/templates/node--directory-record--full.html.twig

```diff
@@ -14,21 +14,31 @@
   {% if content.field_phone_number|render %} 
   <div>
     <h3>Phone Number</h3>
-    <p>{{content.field_phone_number}}</p>
+    <p>
+      
```

Comment:
Just curious -- is there a reason for the additional blank lines here?

Reply:
No just my vscode settings, I'll remove them!